web/client/components/mapcontrols/annotations/AnnotationsEditor.jsx

```diff
@@ -317,6 +326,8 @@ class AnnotationsEditor extends React.Component {
                             onStartDrawing: this.props.onStartDrawing,
                             disabled: !this.props.config.multiGeometry && this.props.editing && this.props.editing.features && this.props.editing.features.length,
                             drawing: this.props.drawing,
+                            defaultStyles: this.props.defaultStyles,
+                            defaultPointType: this.props.defaultPointType,
```

Comment:
should this be changed like this?
```suggestion
                            defaultPointType: this.getConfig().defaultPointType,
```